app/views/admin/enterprises/form/_users.html.haml

```diff
@@ -17,27 +10,25 @@
       %a= t('admin.whats_this')
   .eight.columns.omega
     - if full_permissions
-      = f.hidden_field :owner_id, class: "select2 fullwidth", 'user-select' => 'Enterprise.owner'
+      = f.hidden_field :owner_id, class: "select2 fullwidth", 'user-select' => 'Enterprise.owner', 'ng-model' => 'Enterprise.owner'
     - else
       = owner_email
 
 .row
   .three.columns.alpha
-    = f.label :email, t('.notifications')
+    =f.label :user_ids, t('.contact')
     - if full_permissions
       %span.required *
-    .with-tip{'data-powertip' => t('.notifications_tip')}
+    %div{'ofn-with-tip' => t('.contact_tip')}
       %a= t('admin.whats_this')
+
   .eight.columns.omega
     - if full_permissions
-      = f.text_field :email, { placeholder: t('.notifications_placeholder'), "ng-model" => "Enterprise.email" }
+      %select.select2.fullwidth{id: 'receives_notifications_dropdown', name: 'receives_notifications', ng: {model: 'receivesNotifications', init: "receivesNotifications = '#{@notification_user}'"}}
+        %option{ng: {repeat: 'user in Enterprise.users', selected: "user.id == #{@notification_user}", hide: '!user.confirmed'}, value: '{{user.id}}'}
+          {{user.email}}
```

Comment:
You've gone to the trouble of (serialising the contact user for the enterprise)[https://github.com/openfoodfoundation/openfoodnetwork/pull/1720/files#diff-bd9e70456365d01bfc035ddbb2b37bd8R10], so perhaps it would be better to use it here rather than using a rails instance variable?